@types/interfaces.d.ts

```diff
@@ -0,0 +1,19 @@
+import { ContextMenuElement } from '../src/vaadin-context-menu.js';
+
+export interface ContextMenuItem {
+  text?: string,
+  component?: string | HTMLElement,
+  disabled?: boolean,
+  checkable?: boolean,
```

Comment:
The implementation doesn't use `checkable` anywhere. This seems to be a custom property used only in one demo so I think it should not be included here.

Reply:
Thanks, removed.